core/src/main/scala/org/apache/spark/sql/rapids/tool/ClassWarehouse.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.tool
+
+import java.util.Date
+
+import org.apache.spark.resource.{ResourceInformation, ResourceProfile}
+
+class ExecutorInfoClass(val executorId: String, _addTime: Long) {
```

Comment:
Question: Is the information in this class can also be retrieved in the Q tool? then why did not we do that already?